src/index.html

```diff
@@ -9,6 +9,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="nav__wraper">
+        <div class="nav__icon">
+          <a href="#Home">
+            <img src="images/logo.png" alt="Moyo icon">
+          </a>
+        </div>
+        <nav class="navigation">
+          <ul class="nav__list">
+            <li class="nav__item"><a href="#apple" class="nav__link is-active">Apple</a></li>
+            <li class="nav__item"><a href="#samsung" class="nav__link">Samsung</a></li>
+            <li class="nav__item"><a href="#smartphones" class="nav__link">Smartphones</a></li>
+            <li class="nav__item">
```

Comment:
Replace '&amp;' with '&amp;' in the href attribute for proper encoding

Reply:
Guess, it means to replace '#' with '&', right? But why?